.eslintrc

```diff
@@ -48,6 +48,10 @@
     "no-throw-literal": 2,
     "no-trailing-spaces": 2,
     "no-unused-expressions": 0,
+    "no-unused-vars": [2, {
+      "argsIgnorePattern": "^var_|opt_|unused_|_",
```

Comment:
so, the google-camelcase plugin unfortunately starts erroring out if we used `unused_`, are we ok with doing `unusedArg` or `var_unused_`